server/server.go

```diff
@@ -92,10 +95,10 @@ type Server struct {
 	TaskCheckScheduler *taskcheck.Scheduler
 	MetricReporter     *MetricReporter
 	SchemaSyncer       *schemasync.Syncer
-	BackupRunner       *BackupRunner
+	BackupRunner       *backuprun.BackupRunner
```

Comment:
*backup.Runner?

Reply:
backup is a widely used variable for api.Backup. Changed to backuprun.Runner and other places too.